### DISCUSSION TITLE
A bug in the selectsheet function

### DISCUSSION THREAD

**Discussion by mahuifa** [QtExcel/QXlsx#213] [General]
If you use selectSheet to set Sheet1 as the active sheet, and then use the moveSheet function to move Sheet1 from position 1 to position 2, and then query the active sheet, you will find that it is the sheet at position 1, not Sheet1 that has been moved to position 2, I'm not sure Is this a bug or did the developers do it on purpose

**Comment by j2doll** [comment]
Dear @mahuifa 

Sorry for late reply.

Can I see the source code you mentioned?
